app/src/main/res/layout/activity_set_location.xml

```diff
@@ -5,6 +5,12 @@
     android:layout_width="match_parent"
     android:layout_height="match_parent">
 
+    <fragment
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:id="@+id/google_map"
+        android:name="com.google.android.gms.maps.SupportMapFragment" />
+
     <EditText
```

Comment:
I think we might want some kind of white layout behind this EditText so the text is not just floating on the map.